setup.php

```diff
@@ -239,3 +244,115 @@ function plugin_fields_checkFiles($force = false) {
+                        $tmp_field['label'] = PluginFieldsLabelTranslation::getLabelFor($field);
+                        $tmp_field['xml_node'] = strtoupper($field['name']);
+                        $tmp_field['type']  = $field['type'];
+                        $tmp_field['ranking'] = $field['ranking'];
+                        $tmp_field['default_value'] = $field['default_value'];
+                        $tmp_field['mandatory'] = $field['mandatory'];
+                        $tmp_field['possible_value'] = "";
+
+                        switch ($field['type']) {
+                           case 'dropdown':
+                              $obj = new $itemtype;
+                              $obj->getEmpty();
+
+                              $dropdown_itemtype = PluginFieldsDropdown::getClassname($field['name']);
+                              $tmp_field['xml_node'] = strtoupper(getForeignKeyFieldForItemType($dropdown_itemtype));
+
+                              $dropdown_obj = new $dropdown_itemtype();
+                              $dropdown_datas = $dropdown_obj->find();
+                              $datas = [];
+                              foreach ($dropdown_datas as $key => $value) {
+                                 $items = [];
+                                 $items['id'] = (int)$value['id'];
+                                 $items['value'] = $value['name'];
+                                 $datas[] = $items;
+                              }
+                              $tmp_field['possible_value'] = $datas;
+                              break;
+                           case 'yesno':
+                              $datas = [];
+                              $datas["0"]['id'] = 0;
+                              $datas["0"]['value'] = __('No');
+                              $datas["1"]['id'] = 1;
+                              $datas["1"]['value'] = __('Yes');
+                              $tmp_field['possible_value'] = $datas;
+                              break;
+                           case 'dropdownuser':
+                              $datas = Dropdown::getDropdownUsers(['is_active' => 1,'is_deleted' => 0], false);
+                              $tmp_field['possible_value'] = $datas['results'];
+                              break;
+                        }
+                        $yaml_conf['container'][$container['id']."-".$itemtype]["fields"][] = $tmp_field;
+                     }
+                  }
+               }
+            }
+         }
+      }
+   }
+
+   if (count($yaml_conf)) {
+      $dump =   Yaml::dump($yaml_conf, 10);
+      $filename = GLPI_TMP_DIR."/fields_conf.yaml";
+      file_put_contents($filename, $dump);
+      return true;
+   }
+
+   return false;
+}
+/**
+ * Check configuration process
+ *
+ * @param boolean $verbose Whether to display message on failure. Defaults to false
+ *
+ * @return boolean
+ */
+function plugin_fields_check_config($verbose = false) {
+   return true;
+}
```

Comment:
This method is not mandatory anymore in GLPI 9.5.
```suggestion
```

Reply:
done